### DISCUSSION TITLE
Newpipe fast foward no indication bar?

### DISCUSSION THREAD

**Discussion by cjazz2014** [TeamNewPipe/NewPipe#10559] [Questions]
Not sure if this is a bug when using version 0.25.2? When using the fast forward option with a video playing there is no time bar displaying at the bottom of the screen (like youtube) makes navigating difficult. Nvidia Shield Pro

**Comment by opusforlife2** [comment]
Did it work in a previous version?

**Reply by cjazz2014** [reply]
Didn't try the previous version.

**Reply by opusforlife2** [reply]
Oh. Your phrasing made it sound like it was a new bug in 0.25.2.

Could you show a screenshot of this in both apps?

**Reply by cjazz2014** [reply]
Apologies for that. It's difficult to show a screenshot of the problem on the Newpipe app because there is nothing on the screen when using the fast forward option i.e no bottom bar.

Attached is a youtube screenshot which shows the time bar when fast forwarding.

It may be just an app setting that needs configuring to get this bar showing on fast forward?
![youtube](https://github.com/TeamNewPipe/NewPipe/assets/10180651/93be6049-bc44-40e5-a08c-35ab127af108)

**Reply by opusforlife2** [reply]
Both options are available. If you tap once to show the seekbar, it will remain visible while you fast-forward or rewind. If you start double tapping right away, the seekbar isn't shown.

**Reply by cjazz2014** [reply]
Tried tapping once, still no seekbar displaying. I tried using an alternative external video player (VLC) this does display a time in seconds on right hand side of the screen for fast forwarding in seconds, no seekbar though.
I have tried the option of "use fast inexact seek" which does show a small seekbar on the top right, although this is unusable because the increments are too high.
I was hoping for a similar seekbar like youtube's default one, so the problem might be associated with the newpipe videoplayer limitations.

**Reply by opusforlife2** [reply]
> Tried tapping once, still no seekbar displaying

Do you also not see the play/pause button, resolution dropdown, etc? Or only the seekbar is invisible?

**Reply by cjazz2014** [reply]
No play/pause button and seekbar are displaying, it's just video only on fast forward.
One thing that may be relevant, when I pause the clip the seekbar is displaying. It's when I use the fast-forward button nothing is displayed. Photo attached shows when the video is paused.
![IMG_4552](https://github.com/TeamNewPipe/NewPipe/assets/10180651/51a9730d-adc2-4d26-b085-0e58b49e035b)

**Reply by opusforlife2** [reply]
Oh. Then this must be an Android TV-specific thing, since I'm assuming you fast-forward or rewind directly using arrow keys on your remote or something. You don't need to interact with the UI at all.

If this is the case, could you open a new feature request for this?
